__tests__/app.test.js

```diff
@@ -185,4 +186,67 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("POST-/api/articles/:article_id/comments", () => {
+    test("201-endpoint accepts a body containing a username and comment body, adds an entry to the comments table and responds with the newly added comment", () => {
+      const requestBody = {
+        username: "butter_bridge",
+        commentBody: "It hasn't got better",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send({
+          username: "butter_bridge",
+          body: "It hasn't got better",
+        })
+        .expect(201)
+        .then((results) => {
+          expect(results.body.comment).toMatchObject({
+            body: "It hasn't got better",
+            votes: 0,
+            author: "butter_bridge",
+            article_id: 1,
+            created_at: expect.any(String),
+          });
+        });
```

Comment:
You could add in another test for a status 201, that ignores unnecessary properties 💡